xml/System.IO.Compression/BrotliDecoder.xml

```diff
@@ -52,12 +52,12 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesConsumed">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The byte read-only span containing the compressed data.</param>
+        <param name="destination">When this method returns, a byte span containing the decompressed data.</param>
+        <param name="bytesConsumed">The total number of bytes that were read from <paramref name="source" />.</param>
+        <param name="bytesWritten">The total number of bytes that were written in the <paramref name="destination" />.</param>
+        <summary>Decompresses data that was compressed using the Brotli algorithm.</summary>
+        <returns>One of the enumeration values that can be returned from span-based operations that support processing of input contained in multiple discontiguous buffers.</returns>
         <remarks>To be added.</remarks>
```

Comment:
We should add remarks for what happens for the semantics of what it means for different `OperationStatus` being returned.

Reply:
Does this sound good?:

```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

The return value can be as follows:
- <xref:System.Buffers.OperationStatus.Done?displayProperty=nameWithType>: Processing of the entire input span succeeded.
- <xref:System.Buffers.OperationStatus.DestinationTooSmall?displayProperty=nameWithType>: There is not enough space in the output span to write the decoded input.
- <xref:System.Buffers.OperationStatus.NeedMoreData?displayProperty=nameWithType>: `isFinalBlock` is `false` and the input is not a multiple of 4. Otherwise, the partial input is considered <xref:System.Buffers.OperationStatus.InvalidData>.
- <xref:System.Buffers.OperationStatus.InvalidData?displayProperty=nameWithType>: The input contains bytes outside of the expected base 64 range, has invalid or more than two padding characters, or is incomplete (that is, not a multiple of 4) and `isFinalBlock` is `true`.
           ]]></format>
         </remarks>
```

Reply:
There is no is final block parameter in this API and none of the base-64 specific details make sense here. I wouldn't copy it as is, but rather update it to be context specific to the decompression API.

Take a look at the implementation/tests to reason about the behavior.
https://source.dot.net/#System.IO.Compression.Brotli/System/IO/Compression/dec/BrotliDecoder.cs,f2d7661515181718

Reply:
```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

The return value can be as follows:
- <xref:System.Buffers.OperationStatus.Done?displayProperty=nameWithType>: `source` was successfully and completely decompressed into `destination`.
- <xref:System.Buffers.OperationStatus.DestinationTooSmall?displayProperty=nameWithType>: There is not enough space in `destination` to decompress `source`.
- <xref:System.Buffers.OperationStatus.NeedMoreData?displayProperty=nameWithType>: The decompression action is partially done at least one more byte is required to complete the decompression task. This method should be called again with more input to decompress.
- <xref:System.Buffers.OperationStatus.InvalidData?displayProperty=nameWithType>: The data in `source` is invalid and could not be decompressed.

           ]]></format>
         </remarks>
```